app/javascript/packs/tta-report.js

```diff
@@ -24,3 +24,29 @@ document.querySelector("#js-close-tta-form").addEventListener('click', event =>
   event.preventDefault()
   closeForm()
 })
+
+document.querySelectorAll(".js-edit-tta").forEach((el) => {
+  el.addEventListener('click', event => {
+    event.preventDefault()
+
+    // hide details
+    const displayId = event.currentTarget.dataset.displayId
+    document.querySelector("#tta-activity-show-" + displayId).classList.add("display-none")
+    // show form
+    const form = document.querySelector("#edit-tta-activity-" + displayId)
+    form.classList.remove("display-none")
```

Comment:
[suggestion] it'd be nice if the edit link also focused the text field, like linking a new report does.

Reply:
yes great call